collections/made-in-iran/index.md

```diff
@@ -1,23 +1,25 @@
 ---
 items:
- - jadijadi/bestoon
+ - jadijadi/linuxandlife
  - jadijadi/justforfun
- - mr-hashemi/mr-hashemi
- - pashmaklang/pashmak
- - sobhe/moratab
- - PyFarsi/pyabr
- - jamedadi/yummy
- - sobhe/hazm
+ - jadijadi/bestoon
+ - KambizAsadzadeh/Project-Template
  - rastikerdar/vazirmatn
  - mohebifar/made-in-iran
  - Hameds/APIs-made-in-Iran
+ - sobhe/moratab
+ - jamedadi/yummy
+ - sobhe/hazm
+ - mr-hashemi/mr-hashemi
+ - PyFarsi/pyabr
+ - pashmaklang/pashmak
```

Comment:
We aren't accepting re-order PRs at this time.

Reply:
`KambizAsadzadeh/Project-Template` is a new one.

Reply:
`jadijadi/linuxandlife` is a new one too.

Reply:
sounds good, just append them to the end and we can review

Reply:
There are the same authors, It's why I posted that at the top. By the way, I did what you asked.